.github/workflows/npm-release.yml

```
@@ -0,0 +1,63 @@
+name: npm-release
+
+on:  # yamllint disable-line rule:truthy
+  workflow_dispatch:
+    inputs:
+      releaseVersion:
+        type: choice
+        description: "Select version increment type (follows Semantic Versioning)"
+        required: true
+        options:
+          - patch
+          - minor
+          - major
+      app-path:
+        type: choice
+        description: "Select the node service to release"
+        required: true
+        options:
+          - refarch-frontend # Add or modify frontend paths according to your project structure
+jobs:
+  release:
+    name: Release
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write # to be able to publish a GitHub release
+      issues: write # to be able to comment on released issues
+      pull-requests: write # to be able to comment on released pull requests
+      id-token: write # to enable use of OIDC for npm provenance
+    outputs:
+      ARTIFACT_NAME: ${{ steps.npm_release.outputs.ARTIFACT_NAME }}
+      ARTIFACT_VERSION: ${{steps.npm_release.outputs.ARTIFACT_VERSION}}
+    steps:
+      - name: NPM Release
```

```suggestion
      - name: Npm Release
```

or remove the name. Other steps don't use this attribute.

---

ja man braucht nur die id für den output

---

```suggestion
      - name: npm release
```
same here